doc/source/whatsnew/v0.24.0.txt

```diff
@@ -607,6 +607,7 @@ Indexing
 - Fixed ``DataFrame[np.nan]`` when columns are non-unique (:issue:`21428`)
 - Bug when indexing :class:`DatetimeIndex` with nanosecond resolution dates and timezones (:issue:`11679`)
 - Bug where indexing with a Numpy array containing negative values would mutate the indexer (:issue:`21867`)
+- Fix ``OverflowError`` when trying to use 'inf' as ``na_value`` with integer index column (:issue:`17128`)
```

Comment:
can you move to 0.23.5 (rebase on master to see the whatsnew)

Reply:
will do in few minutes

Reply:
@jreback  done.